### ISSUE TITLE
Incorrect README hyperlink for Point-MaDi NeurIPS 2025 Poster

### ISSUE THREAD

**Issue by Zaozzz** [YangParky/Point-MaDi#2]
Hi, thanks for open-sourcing this excellent work.

I noticed a possible issue in the README. The hyperlink attached to:

Point-MaDi: Masked Autoencoding with Diffusion for Point Cloud Pre-training (NeurIPS 2025 Poster)

currently points to:

https://arxiv.org/abs/2302.02318

This link seems inconsistent with the paper title / venue information shown in the README, so it may be the wrong paper link.

It might be helpful to update it to the correct paper page, so readers can access the intended publication directly.

Thanks again for sharing your code.

**Comment by YangParky** [comment]
I've just updated it to the correct paper page. Really appreciate your careful review and for helping improve the README. Thanks again !
